components/coding/profileV2/JoinedDateComponent.tsx

```diff
@@ -6,12 +6,26 @@ import { User } from "./ProfileHeaderComponent";
 export type JoinedDateComponentProps = {
   user: User;
   createdAt: Date;
+  textSize: "small" | "medium" | "large";
 };
 
 export default function JoinedDateComponent({
   user,
   createdAt,
+  textSize,
 }: JoinedDateComponentProps) {
+  let textStyle;
+  switch (textSize) {
+    case "small":
+      textStyle = "ml-2 text-sm mg:text-sm";
```

Comment:
same question with mg on lines 20 and 23

Reply:
+1 to this

Reply:
Fixed!